jib-gradle/jib-gradle.yaml

```diff
@@ -42,6 +42,7 @@ spec:
         --stacktrace --console=plain \
         --init-script=/tekton/home/init-script.gradle \
         -Duser.home=/tekton/home \
+        --gradle-user-home=/tekton/home/.gradle \
```

Comment:
do we need both these values? line 45 and 46?

Reply:
Maybe not, but just wanted to be on the safe side. BTW, `--gradle-user-home` is documented in `man gradle`, so I think this is the right way to do it. Do you think we should remove line 64?

Reply:
I have no idea, I guess it couldn't hurt.